src/main/java/rs/teslaris/core/controller/OrganisationUnitController.java

```diff
@@ -77,12 +76,4 @@ public OrganisationUnitDTO updateOrganisationUnit(
                 organisationUnitId);
         return OrganisationUnitConverter.toDTO(organisationUnit);
     }
-
-    @DeleteMapping("/{organisationUnitId}")
-    @PreAuthorize("hasAuthority('EDIT_ORGANISATION_UNITS')")
-    @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void deleteOrganisationUnit(@PathVariable Integer organisationUnitId) {
-        organisationUnitService.delete(organisationUnitId);
-    }
```

Comment:
I think we should have removing organization units. If there are objects linked to the certain organization unit, the deletion wouldn't be possible, but if not, it should be possible. Basically, the scenario might be that some admin added an organization unit and then changed his/her mind and decide to delete that.

Reply:
Of course, we removed this method because there were no checks implemented but the plan was to add it back properly.

Reply:
Done